include/slam_toolbox/slam_toolbox.hpp

```diff
@@ -135,18 +136,21 @@ class SlamToolbox
   // state
   bool IsPaused(const PausedApplication& app);
 
+  void SaveDataToFile(const std::string& filename);
```

Comment:
`SerializeMapper`, also I dont see this function implemented in the cpp file, where is it?